examples/transformers_text.py

```diff
@@ -243,6 +241,18 @@ def mean_pooling(last_hidden_state: Tensor, attention_mask) -> Tensor:
 val_loader = DataLoader(val_tensor_frame, batch_size=args.batch_size)
 test_loader = DataLoader(test_tensor_frame, batch_size=args.batch_size)
 
+if not args.finetune:
+    text_stype_encoder = LinearEmbeddingEncoder()
+else:
+    model_cfg = TextModelConfig(model=text_encoder, out_channels=768)
+    col_to_text_model_cfg = {
+        col_name: model_cfg
+        for col_name in train_tensor_frame.col_names_dict[
+            torch_frame.text_tokenized]
+    }
+    text_stype_encoder = LinearModelEncoder(
```

Comment:
Rename to `LinearTextModelEncoder`

Reply:
since this seems text-specific.

Reply:
For now I think I can change to `LinearTextModelEncoder` but this encoder actually also works for some other embeddings right?

Reply:
Also could you update the doc-string accordingly?

Reply:
WDYT?

Reply:
Doc-string of `LinearTextModelEncoder` to mention it is for text and takes text embedding model as input.